### PR TITLE
💘 Fix tooltip arrow

### PR DESCRIPTION
**Pull request by ClumsyVlad**

### CODE REVIEW

File: packages/ui/src/common/components/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -72,7 +73,7 @@ export const Tooltip = ({
     onFocus: mouseIsOver,
     onBlur: mouseLeft,
     onMouseEnter: mouseIsOver,
-    onMouseLeave: mouseLeft,
+    // onMouseLeave: mouseLeft,
```
Comment (jodator) [inline packages/ui/src/common/components/Tooltip/Tooltip.tsx:76]: This should be reverted or removed?

File: packages/ui/src/common/components/Tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -94,6 +95,7 @@ export const Tooltip = ({
                 {...attributes.popper}
                 {...popUpHandlers}
                 isTooltipActive={isTooltipActive}
+                forBig={forBig}
```
Comment (jodator) [inline packages/ui/src/common/components/Tooltip/Tooltip.tsx:98]: Maybe we could use the [arrow](https://popper.js.org/docs/v2/modifiers/arrow/) from popper?

Comment (jodator) [inline packages/ui/src/common/components/Tooltip/Tooltip.tsx:98]: Maybe we could use the [arrow](https://popper.js.org/docs/v2/modifiers/arrow/) from popper?

**Review by jodator — COMMENTED** [review]
Maybe you could check the [arrow modifier](https://popper.js.org/docs/v2/modifiers/arrow/)?

File: packages/ui/src/common/components/Tooltip/Tooltip.tsx (hunk 2 above)
Comment (ClumsyVlad) [reply packages/ui/src/common/components/Tooltip/Tooltip.tsx:98]: I've tried, it doesn't work for me :(

**Pull request merged**